materialized_sql_view/__openerp__.py

```diff
@@ -0,0 +1,93 @@
+# =============================================================================
+#                                                                             =
+#    materialized_sql_view module for OpenERP,                                =
+#    Copyright (C) 2013 Anybox (<http://http://anybox.fr>)                    =
+#                         Pierre Verkest <pverkest@anybox.fr>                 =
+#                                                                             =
+#    This file is a part of materialized_sql_view                             =
+#                                                                             =
+#    materialized_sql_view is free software: you can redistribute it and/or   =
+#    modify it under the terms of the GNU Affero General Public License v3 or =
+#    later as published by the Free Software Foundation, either version 3 of  =
+#    the License, or (at your option) any later version.                      =
+#                                                                             =
+#    materialized_sql_view is distributed in the hope that it will be useful, =
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of           =
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the            =
+#    GNU Affero General Public License v3 or later for more details.          =
+#                                                                             =
+#    You should have received a copy of the GNU Affero General Public License =
+#    v3 or later along with this program.                                     =
+#    If not, see <http://www.gnu.org/licenses/>.                              =
+#                                                                             =
+# =============================================================================
+{
+    'name': 'Materialized Sql View',
+    'version': '0.1',
+    'category': 'Tools',
+    'description': """
```

Comment:
Thanks for having created README.rst

Now you just need to remove `description` key from `__openerp__.py`